app/Models/Categories.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Categories extends Model
+{
+    use HasFactory;
+
+    protected $fillable = ['name', 'slug', 'description'];
```

Comment:
Normalmente se coloca no $fillable campos que podem ser atribuidos em massa (através do metodo `fill` ou `create` da model). Nesse teu caso, o slug é um campo que deve ser gerado automaticamente e nunca o usuário pode dizer qual será o valor do slug.
Nesse caso recomendo remover o slug daqui pra ele não ser preenchido sem querer
```suggestion
    protected $fillable = ['name', 'description'];
```